client/debian/control

```diff
@@ -12,7 +12,7 @@ Build-Depends: cargo:native (>= 1.75),
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Uploaders: Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
 Standards-Version: 4.7.0
-XS-Vendored-Sources-Rust: aho-corasick@1.1.3, anyhow@1.0.98, atomic-waker@1.1.2, autocfg@1.4.0, base64@0.22.1, bitflags@2.9.0, bytes@1.10.1, cc@1.2.19, cfg-if@1.0.0, deranged@0.4.0, diff@0.1.13, displaydoc@0.2.5, either@1.15.0, encoding_rs@0.8.35, equivalent@1.0.2, fnv@1.0.7, foreign-types-shared@0.1.1, foreign-types@0.3.2, form_urlencoded@1.2.1, futures-channel@0.3.31, futures-core@0.3.31, futures-sink@0.3.31, futures-task@0.3.31, futures-util@0.3.31, getopts@0.2.21, getrandom@0.2.16, h2@0.4.9, hashbrown@0.15.2, heck@0.5.0, http-body-util@0.1.3, http-body@1.0.1, http@1.3.1, httparse@1.10.1, hyper-rustls@0.27.5, hyper-tls@0.6.0, hyper-util@0.1.11, hyper@1.6.0, icu_collections@1.5.0, icu_locid@1.5.0, icu_locid_transform@1.5.0, icu_locid_transform_data@1.5.1, icu_normalizer@1.5.0, icu_normalizer_data@1.5.1, icu_properties@1.5.1, icu_properties_data@1.5.1, icu_provider@1.5.0, icu_provider_macros@1.5.0, idna@1.0.3, idna_adapter@1.2.0, indexmap@2.9.0, indoc@2.0.6, ipnet@2.11.0, itertools@0.14.0, itoa@1.0.15, lazy_static@1.5.0, libc@0.2.172, litemap@0.7.5, log@0.4.27, memchr@2.7.4, memoffset@0.9.1, mime@0.3.17, mio@1.0.3, native-tls@0.2.14, num-conv@0.1.0, once_cell@1.21.3, openssl-macros@0.1.1, openssl-probe@0.1.6, openssl-sys@0.9.107, openssl@0.10.72, percent-encoding@2.3.1, pin-project-lite@0.2.16, pin-utils@0.1.0, pkg-config@0.3.32, powerfmt@0.2.0, pretty_assertions@1.4.1, proc-macro2@1.0.95, pyo3-build-config@0.24.2, pyo3-ffi@0.24.2, pyo3-macros-backend@0.24.2, pyo3-macros@0.24.2, pyo3@0.24.2, quote@1.0.40, regex-automata@0.4.9, regex-syntax@0.8.5, regex@1.11.1, reqwest@0.12.15, ring@0.17.14, rustls-pemfile@2.2.0, rustls-pki-types@1.11.0, rustls-webpki@0.103.1, rustls@0.23.26, ryu@1.0.20, serde@1.0.219, serde_derive@1.0.219, serde_json@1.0.140, serde_urlencoded@0.7.1, shlex@1.3.0, simple_test_case@1.2.0, slab@0.4.9, smallvec@1.15.0, smbios-lib@0.9.2, socket2@0.5.9, stable_deref_trait@1.2.0, subtle@2.6.1, syn@2.0.100, sync_wrapper@1.0.2, synstructure@0.13.1, target-lexicon@0.13.2, time-core@0.1.4, time-macros@0.2.22, time@0.3.41, tinystr@0.7.6, tokio-macros@2.5.0, tokio-native-tls@0.3.1, tokio-rustls@0.26.2, tokio-util@0.7.15, tokio@1.45.0, tower-layer@0.3.3, tower-service@0.3.3, tower@0.5.2, tracing-core@0.1.33, tracing@0.1.41, try-lock@0.2.5, unicode-ident@1.0.18, unicode-width@0.1.14, unindent@0.2.4, untrusted@0.9.0, url@2.5.4, utf16_iter@1.0.5, utf8_iter@1.0.4, vcpkg@0.2.15, want@0.3.1, write16@1.0.0, writeable@0.5.5, yansi@1.0.1, yoke-derive@0.7.5, yoke@0.7.5, zerofrom-derive@0.1.6, zerofrom@0.1.6, zeroize@1.8.1, zerovec-derive@0.10.3, zerovec@0.10.4
```

Comment:
Can you wrap-and-sort this please (run the tool)

Reply:
I believe I already did that, but I’ve just checked that when I run the `wrap-and-sort` command, it doesn't change anything (but used to change the order of dependencies for example, so it works)

Reply:
Ah I expected it to make that long line different. Oh well.